packages/@lwc/engine/src/framework/main.ts

```diff
@@ -22,3 +22,6 @@ export { readonly } from './readonly';
 export { deprecatedBuildCustomElementConstructor as buildCustomElementConstructor } from './wc';
 
 export { setFeatureFlag, setFeatureFlagForTest } from '@lwc/features';
+
+// Types -------------------------------------------------------------------------------------------
+export type { Renderer } from './vm';
```

Comment:
why is this here?

Reply:
This is type is here in preparation for the split. If the `@lwc/engine-dom` and the `@lwc/engine-server` injects a renderer we will need to expose this interface out of the `@lwc/engine-core` so the different packages can leverage it to be type-safe.